lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1573,6 +1573,23 @@ SPIRVValue *LLVMToSPIRV::transIntrinsicInst(IntrinsicInst *II,
     return BM->addExtInst(transType(II->getType()),
                           BM->getExtInstSetId(SPIRVEIS_OpenCL), OpenCLLIB::Sqrt,
                           {transValue(II->getOperand(0), BB)}, BB);
```

Comment:
`}` closing the sqrt case is missing.

Reply:
AH. That must have snuck through during the merge. Thanks for the catch

Reply:
Fixed